White Toggle Button

german-e-mas opened the pull request:
This PR addresses part of https://github.com/space-ros/spaceros.org/issues/16

The sidenav toggle button is now white and has a _difference_ blend, making it stand out over any backgrounds.

Can you ptal @mjeronimo ?

---

## GIF

![spaceROS-toggle-btn-002](https://user-images.githubusercontent.com/14120807/206537329-d272f110-606e-4bfb-bc04-2a56ccf877a2.gif)

mjeronimo reviewed (approved):
That looks much better!

german-e-mas commented:
Marking as a WIP - There is an issue with the rest of the site:

![Screenshot from 2022-12-08 15-12-48](https://user-images.githubusercontent.com/14120807/206533136-6b4f73f6-e994-4a50-9b8f-d22093267627.png)

Trying to come up with a solution that solves both scenarios. FYI @mjeronimo .

german-e-mas commented:
> Marking as a WIP - There is an issue with the rest of the site:
> 
> ![Screenshot from 2022-12-08 15-12-48](https://user-images.githubusercontent.com/14120807/206533136-6b4f73f6-e994-4a50-9b8f-d22093267627.png)
> 
> Trying to come up with a solution that solves both scenarios. FYI @mjeronimo .

Addressed in https://github.com/mjeronimo/pelican-fh5co-marble/pull/6/commits/f0a175d43b5dc6cc528ebb51ea6ecd5134ef7377 - A blend mode can now make it stand out on all pages. I updated the description and GIF. FYI @mjeronimo , in case you want to give it another look.

mjeronimo reviewed (approved):
Nice.
